### DISCUSSION TITLE
Update with IN operator gets error "Provided Json, expected String"

### DISCUSSION THREAD

**Discussion by dk253** [prisma/prisma#8704] [Q&A] [answered]
New to Prisma, but it looks VERY promising!

I am trying to run an update query:

```ts
prisma.tbl.update({
  "where": {
    "id": {
      "IN": [
        "Saqui",
        "Clementine",
        "Bob"
      ]
    }
  },
  "data": {
    "readAt": "2021-08-11T17:49:08-07:00"
  }
})
```

But I'm getting an error no matter what I do:

```
{
  where: {
    id: {
      IN: [
        'Saqui',
        'Clementine',
        'Bob'
      ]
    }
    ~~~~~~~~~~~~~~~~~
  },
  data: {
    readAt: '2021-08-11T17:49:08-07:00'
  }
}

Argument id: Got invalid value 
{
  IN: [
    'Saqui',
    'Clementine',
    'Bob'
  ]
}
on prisma.updateOnecochContactChats. Provided Json, expected String.
```

What am I doing wrong?  Thanks!

**Comment by ryands17** [comment, marked as answer]
@dk253 You need to use `in` instead of `IN`. Have a look at the supported operators [here](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#filter-conditions-and-operators).

**Reply by dk253** [reply]
Thanks, @ryands17!  You are right but not completely.  I had tried the in and the IN.  Both failed.  But when I changed to "updateMany" instead of "update", the "in" started working!  

Thanks for your help.

**Comment by dk253** [comment]
Changed from update to updateMany and IN to in.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
